urllib3/_collections.py

```diff
@@ -167,7 +167,7 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
-    if not PY3: # Python 2
+    if not PY3:  # Python 2
```

Comment:
do we want to keep these comments on the same line as `PY3`?  They dont add much.